### PR TITLE
Add Xcode plugin named XNatalie to generate storyboards code

### PR DESCRIPTION
**Pull request by phimage**
in Xcode Product menu, added a Natalie menu
- Generate to launch code generation
- Enable generate when saving : to save each time a user do File>Save 
- the last menu to change natalie.swift path

Feel free to reject the pull request, if you prefer I will make an independent project

### CODE REVIEW

File: XNatalie/XNatalie/NataliePlugin.swift [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        addObservers()
+    }
+
+    deinit {
+        removeObservers()
+    }
+    
+    // MARK: config
+    
+    static let pluginEnabledString = "NathalieEnabled"
```
Comment (krzyzanowskim) [inline XNatalie/XNatalie/NataliePlugin.swift]: "Nathalie" ? ;-)

**Comment by krzyzanowskim** [comment]
I love the idea, thanks! will merge ASAP, can you just please change "Nathalie" -> "Natalie" ?

**Comment by phimage** [comment]
Done
Remove also a duplicate file

**Comment by krzyzanowskim** [comment]
Can't build the project
screen shot 2015-05-15 at 01 19 22

**Comment by krzyzanowskim** [comment]
@phimage  Can't make it work with Xcode 6.3.2, it crashes at startup.

**Comment by phimage** [comment]
I see you make a commit with the correct search path
`/Applications/Xcode.app/Contents/SharedFrameworks,`
`/Applications/Xcode.app/Contents/Frameworks,`
 (Xcode add my custom path automatically and i don't see it ;()

This fix the issues? if not maybe the `$(inherited)`, must not be removed

I can't access a mac with Xcode until sunday or monday (I think I have Xcode 6.3.2 or 6.3.1) to test
Sorry for the inconvenient

**Comment by phimage** [comment]
6.3.2 is now official
unfortunately plugins must be updated with xcode version ID in info.plist to be loaded
I use this plugin template as reference
https://github.com/kattrali/Xcode-Plugin-Template/commit/9af366334a8591948d26b72e814926a4a38b7263

moreover there is a bug, xcode menu creation is postponed
I will create new pull request with fix

**Comment by krzyzanowskim** [comment]
merged

**Pull request merged**